cmd/ccboc/main.go

```diff
@@ -222,6 +304,11 @@ func main() {
 				Name:  "logG",
 				Usage: "specifies the logG value when creating a calculation.",
 			},
+			&cli.StringFlag{
+				Name:    "path",
```

Comment:
argument named `path` is unclear what it does. Let us have a better name, like `results-download-path`